lib/Echidna/Campaign.hs

```diff
@@ -51,6 +51,8 @@ instance MonadThrow m => MonadThrow (RandT g m) where
 instance MonadCatch m => MonadCatch (RandT g m) where
   catch = liftCatch catch
 
+type MutationConsts = (Integer, Integer, Integer)
```

Comment:
let's turn this into an actual lensible product type:

```haskell
data MutatorWeights = MutatorWeights { _mutatorXXX :: Integer, ... }
makeLenses ''MutatorWeights
```

Reply:
Let's keep this a list, while we experiment which mutator is actually useful